app/script/calling/entities/CallEntity.js

```diff
@@ -63,13 +63,7 @@ z.calling.entities.CallEntity = class CallEntity {
     const loggerName = 'z.calling.entities.CallEntity';
     this.callLogger = new z.telemetry.calling.CallLogger(loggerName, this.id, z.config.LOGGER.OPTIONS, this.messageLog);
 
-    this.callLogger.info({
-      data: {
-        default: [this.id],
-        obfuscated: [this.callLogger.obfuscate(this.id)],
-      },
-      message: `Created new call entity in conversation {0}`,
-    });
+    this.callLogger.info(`Created new call entity in conversation ${this.id}`);
```

Comment:
Seems we have lost obfuscation here

Reply:
According to @pwnsdx obfuscation of IP addresses and UUID is handled by default. We only need to pass in the object in cases where we replace names with IDs. Not exactly obvious. We need to see how to improve the documentation and handling of the call logger.